package.json

```diff
@@ -38,7 +38,10 @@
         "wayou.vscode-todo-highlight",
         "wix.vscode-import-cost",
         "YoshinoriN.current-file-path",
-        "Zignd.html-css-class-completion"
+        "Zignd.html-css-class-completion",
+        "esbenp.prettier-vscode",
+        "jkjustjoshing.vscode-text-pastry",
+        "tombonnike.vscode-status-bar-format-toggle"
```

Comment:
It’s not alphabetical 😱😜

Reply:
Its not a big deal, I like to start of organised, I might not end up there in a few months 😂